pkgs/development/interpreters/spidermonkey/common.nix

```diff
@@ -55,6 +61,24 @@ stdenv.mkDerivation rec {
     zlib
   ];
 
+  patches = lib.optional (lib.versionOlder version "91") [
```

Comment:
The 78 expression had an ordering of attributes `patches` just after `src`, which is slightly more common, see https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/attribute-ordering.md. It might be nice to preserve it.